static/templates/stream_description.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if rendered_description}}
+<span class="sub-stream-description rendered_markdown">
+    {{rendered_markdown rendered_description}}
+</span>
+{{else}}
+<span class="sub-stream-description no-description">
+    This stream does not have a description configured.
```

Comment:
This string should be tagged for translation.

Reply:
Also, can you explain why this approach is better than just changing the text in the `data-no-description` thing we had before?

Reply:
`data-no-description` approach appears to be partially broken probably after #18872. It shows nothing for a stream with no description and shows `No description.` only after some edits to the description.
I had tried for about 2hrs to solve it using `data-no-description` approach but couldn't. So I thought of this approach, it feels better in the sense that it is more stable to any refactors we may do in that area.

Reply:
OK, sounds good.